src/backend/opengl/opengl_display.cpp

```diff
@@ -188,10 +187,14 @@ void c_OpenGLDisplay::OnPaint(wxPaintEvent&)
 {
     wxPaintDC dc(m_GLCanvas.get());
 
-    glBindFramebuffer(GL_DRAW_FRAMEBUFFER, 0);
-    const auto s = m_GLCanvas->GetSize();
-    glViewport(0, 0, s.GetWidth(), s.GetHeight());
+    SetGLContextOnGLCanvas();
 
+    glBindFramebuffer(GL_DRAW_FRAMEBUFFER, 0);
+    const auto canvasSize = m_GLCanvas->GetSize();
+    glViewport(0,
```

Comment:
Please rather use this formatting (also in other cases):
```C++
glViewport(
    0,
    0,
    canvasSize.GetWidth() * m_ScaleFactor,
    canvasSize.GetHeight() * m_ScaleFactor
);
```
```

Reply:
FYI, this is Xcode's built-in formatter ;)